coinbase_curl_check.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Direct Coinbase API check via curl
+
+source .env
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)